modules/edu.gemini.epics.acm/src/main/java/edu/gemini/epics/acm/CaCommandMonitorImpl.java

```diff
@@ -130,4 +130,18 @@ public synchronized void completeFailure(Exception cause) {
         }
     }
 
+    public synchronized void completePause() {
+        lock.lock();
```

Comment:
Should the `unlock` be done on a `try/finally` block? in case some call throws?

Reply:
`unlock` doesn't throw checked exceptions.